README.md

```diff
@@ -216,8 +215,8 @@ The interface looks like this:
 
 ```ts
 export interface ThrottlerStorage {
-  getRecord(key: string): Promise<number[]>;
-  addRecord(key: string, ttl: number): Promise<void>;
+  storage: Record<string, { totalHits: number; timeToExpire: number }>;
+  addRecord(key: string, ttl: number): Promise<{ totalHits: number; timeToExpire: number }>;
```

Comment:
Yeah, we can make an interface for `{ totalHits, timeToExpire }`. Also make sure to export it for the sake of extension packages

Reply:
I've updated this part as well as other part of the README. Have a look again and resolve this is you're satisfied.